easybuild/easyconfigs/p/python-xxhash/python-xxhash-3.1.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'python-xxhash'
+version = '3.1.0'
+
+homepage = 'https://github.com/ifduyue/python-xxhash'
+description = """Python bindings for xxHash. xxHash is an extremely fast
+non-cryptographic hash algorithm, working at RAM speed limit."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('xxHash', '0.8.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('xxhash', version, {
+        'checksums': ['ac21b1e21dc6fdfee9a57b53f4777539d53a84f2e1546a3f802f159f9966bdc1'],
+        'preinstallopts': 'XXHASH_LINK_SO=1',  # use xxHash library from EB
```

Comment:
@lexming should we add this to python-xxhash-3.2.0-GCCcore-12.2.0.eb as well?

Reply:
@lexming good point, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19432